x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -21,7 +21,7 @@ export function registerIntegrations(
     }),
     description: i18n.translate('xpack.maps.registerIntegrations.gdal.integrationDescription', {
       defaultMessage:
-        'Upload shapefiles and ingest from relational databases such as PostGIS or Oracle Spatial with GDAL.',
+        'Upload shapefiles and ingest from relational databases such as PostGIS or Oracle Spatial with GDAL and Elastic Maps.',
```

Comment:
How about "Upload shapefiles and ingest from relational databases such as PostGIS or Oracle Spatial with GDAL. [file upload, geo, maps]". I don't think there is a natural way to work these keywords into the description so just appending them at the end is better until https://github.com/elastic/package-spec/issues/209 is resoled